### PR TITLE
`mean`: add process graph, clarify null arrays

### PR DESCRIPTION
**Pull request by m-mohr**
Two changes I'd like to get some reviews for:
1. Add a process graph as "alias" #137 - did I miss anything?
    image
    Especially wondering about whether sum and divide are actually called/executed for an an empty array as this would lead to a DivisionByZero exception although it strictly shouldn't throw.
2. Clarify how arrays with only null values are handled. That came up when implementing the alias. Does this work for all?

### CODE REVIEW

**Comment by soxofaan** [comment]
> Especially wondering about whether sum and divide are actually called/executed for an an empty array

I'd think this is a backend-side implementation detail, related to how short-circuiting of logical operations and alike are handled.

**Comment by m-mohr** [comment]
Yes, it should although it can lead to inconsistencies between back-ends, which is not desirable.

**Comment by m-mohr** [comment]
@soxofaan No, `condition` is either `true`, `null` or a process. We only pass true (count all elements) or null (count only non-null elements) in this case.

**Comment by soxofaan** [comment]
indeed, but is `count_condition` a parameter (I don't think so)? Or is it  a node, in which case it should be `"from_node": "count_condition"` instead of `"from_parameter": "count_condition"`?

File: mean.json [hunk 1]
```diff
@@ -86,5 +96,71 @@
...
+            }
+        },
+        "count": {
+            "process_id": "count",
+            "arguments": {
+                "data": {
+                    "from_parameter": "data"
+                },
+                "condition": {
+                    "from_parameter": "count_condition"
```
[the remark this answers is not in the document]
Comment (m-mohr) [reply mean.json:118]: You are right, this should be from_node instead of from_parameter.

[the remark this answers is not in the document]
Comment (m-mohr) [reply mean.json]: Fixed. Interesting, that the Web Editor still showed the correct graph though.

**Pull request merged**